GETTING-STARTED.md

```diff
@@ -0,0 +1,329 @@
+# PatternFly React Getting Started
+
+This README is intended to further detail some of the steps mentioned in the PatternFly React [Contributing Guide](https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#code-consistency) and help new contributors getting started with PatternFly React.
+
+## Learning React
+
+If you haven't already, we highly recommend taking a look at the [React Documentation site](https://reactjs.org/). This contains up to date information on React from the project's core maintainers at Facebook.
+
+We also have a few documents started on this subject with some helpful links and descriptions that might help you learn React. Feel free to add your own docs and helpful links here!
+
+- [Notes for React Beginners](https://docs.google.com/document/d/1O773jB-t8NgirkIA5pnPWjmZRRy9iZ02HS35I7EEYOw/edit) by Mike Turley
```

Comment:
This link isn't visible outside of red hat - we should remove it until we can convert it to either the wiki or something public.